src/models/Application.py

```diff
@@ -112,3 +130,38 @@ def update_score(user_id, score):
             raise e
 
         return application
+
+    @staticmethod
+    def get_scored_applications():
+        """Returns all scored applications"""
+        return db.session.query(Application) \
+            .filter(Application.score != 0) \
+            .all()
```

Comment:
can we not load everything into memory, can we do this with a mega SQL query, I'm pretty sure we can using a group by